src/Context.ts

```diff
@@ -1,6 +1,6 @@
 import { DataSource, EntitySchema, ObjectType, Repository } from 'typeorm';
 import { FastifyRequest, FastifyBaseLogger } from 'fastify';
-import { RootSpan } from '@google-cloud/trace-agent/build/src/plugin-types';
+// import { RootSpan } from '@google-cloud/trace-agent/build/src/plugin-types';
```

Comment:
Remove the comments before you merge the PR please :)

Reply:
Should have removed all comments I previously had created